tests/src/test/scala/spire/math/NumberTest.scala

```diff
@@ -1,19 +1,16 @@
 package spire
 package math
 
-import org.scalatest.FunSuite
-
-import org.scalatest.Matchers
 import org.scalacheck.Arbitrary._
-import org.scalatest._
-import prop._
+import org.scalatest.{Matchers, PropSpec}
+import org.scalatest.check.ScalaCheckDrivenPropertyChecks
+import org.scalatest.funsuite.AnyFunSuite
 
-class NumberPropertiesTest extends PropSpec with Matchers with GeneratorDrivenPropertyChecks {
+class NumberPropertiesTest extends PropSpec with Matchers with ScalaCheckDrivenPropertyChecks {
   property("Number.apply(Long)") {
     forAll { (n: Long) => Number(n) shouldBe n }
     forAll { (n: Long) => Number(n) shouldBe SafeLong(n) }
     // we need to do (n - 1).abs to ensure we don't get a negative number
-    forAll { (n: Long) => Number((n - 1).abs) shouldBe Natural((n - 1).abs) }
```

Comment:
Why is this removed?

Reply:
Weird, must have been a `sed` mistake or something. Good catch. I'll double-check the rest to make sure nothing else disappeared.